tuxemon/core/tools.py

```diff
@@ -529,7 +531,10 @@ def number_or_variable(game, value):
     :raises: ValueError
     """
     player = game.player1
-    if value.isdigit():
+    values = value.split("-")
```

Comment:
I believe we decided to change this to a tilde, yes?

Reply:
It's fine either way by me. But yes, I'm also waiting for confirmation to know if I should leave it as is or change that to a tide.

Reply:
As this was the most voted choice, latest commit uses ~ to divide the random range.